server/services/gameLogic.js

```diff
@@ -12,7 +12,7 @@ const statusTypes = {
 };
 
 const rounds = 3;
-const minPlayers = 3;
+const minPlayers = 0;
```

Comment:
Is this just for testing? I thought you needed 3 players.

Reply:
yep just for testing